packages/quantic/cypress/e2e/default-2/search-box/search-box.cypress.ts

```diff
@@ -0,0 +1,738 @@
+          });
+
+          it('should limit the number of suggestions to display to respect the number of suggestions property', () => {
+            const expectedQuerySuggestions = [
+              ...exampleRecentQueries,
+              ...exampleQuerySuggestions,
+            ].slice(0, customNumberOfSuggestions);
+            scope('when loading standalone search box', () => {
+              Expect.displaySearchBoxInput(true, textarea);
+              Expect.displaySearchButton(true);
+            });
+
+            scope('when focusing on the search box input', () => {
+              Actions.focusSearchBox(textarea);
+              cy.wait(InterceptAliases.QuerySuggestions);
+
+              Expect.displaySuggestionList(true);
+              Expect.displayClearRecentQueriesButton(true);
+              Expect.numberOfQuerySuggestions(customNumberOfSuggestions);
+              Expect.querySuggestionsEquals(expectedQuerySuggestions);
+            });
+          });
+        });
+
+        describe('when a the property #disableRecentQueries is set to true', () => {
+          const exampleRecentQueries = ['foo', 'bar'];
+
+          describe('when the local storage contains recent query suggestions', () => {
+            beforeEach(() => {
+              setRecentQueriesInLocalStorage(exampleRecentQueries);
+              visitSearchBox({
+                ...defaultOptions,
+                textarea,
+                disableRecentQueries: true,
+              });
+              mockQuerySuggestions(exampleQuerySuggestions);
+            });
+
+            it('should not display the recent query suggestions', () => {
+              const expectedQuerySuggestions = exampleQuerySuggestions;
+
+              scope('when loading standalone search box', () => {
+                Expect.displaySearchBoxInput(true, textarea);
+                Expect.displaySearchButton(true);
+              });
+
+              scope('when focusing on the search box input', () => {
+                Actions.focusSearchBox(textarea);
+                cy.wait(InterceptAliases.QuerySuggestions);
+
+                Expect.displaySuggestionList(true);
+                Expect.displayClearRecentQueriesButton(false);
+                Expect.numberOfQuerySuggestions(
+                  expectedQuerySuggestions.length
+                );
+                Expect.querySuggestionsEquals(expectedQuerySuggestions);
+              });
+            });
+          });
+
+          describe('when the local storage does not contain recent query suggestions', () => {
+            beforeEach(() => {
+              setRecentQueriesInLocalStorage([]);
+              visitSearchBox({
+                ...defaultOptions,
+                textarea,
+                disableRecentQueries: true,
+              });
+              mockQuerySuggestions(exampleQuerySuggestions);
+            });
+
+            it('should not add the query to the local storage after executing a new search', () => {
```

Comment:
Yea only this test, and we should be mostly just looking at the suggestions that pops up when we click on the searchbox, **maybe** the localStorage, but not required.